### PR TITLE
Doc : Goerli deprecated

### PR DESCRIPTION
**Pull request by PhilippeR26**
## Motivation and Resolution
Starknet Goerli testnet has been shut down yesterday.

## Usage related changes
Documentation has been updated accordingly.
> At the same time, a small update in migrate.md has been made.

## Development related changes

default network has been changed to SEPOLIA.
> data in constants.ts have not been modified, because it has to be modified also for get-starknet and compatibility with L3 (enum -> string).

## Checklist:

- [x] Rebased to the last commit of the target branch (or merged it into my branch)
- [x] All tests are passing

### CODE REVIEW

**Review by tabaktoni — APPROVED** [review]
In general legit

File: www/docs/guides/L1message.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ You have to pay in the L1 an extra fee when invoking `sendMessageToL2` (of cours
 
 ```typescript
 import { SequencerProvider } from 'starknet';
```
Comment (penovicp) [inline www/docs/guides/L1message.md:38]: Replace `SequencerProvider` with `RpcProvider`. I believe this is the last reference in the non-versioned docs.

**Review by penovicp — CHANGES_REQUESTED** [review]

**Comment by tabaktoni** [comment]
We should wait for PR to clean Goerli from repo/codebase and merge both together

File: www/docs/guides/L1message.md (hunk 1 above)
Comment (PhilippeR26) [reply www/docs/guides/L1message.md]: Made.

**Review by ivpavici — APPROVED** [review]

**Review by penovicp — APPROVED** [review]

**Pull request merged**